docs/development_guide.md

```diff
@@ -51,7 +55,56 @@ Note that after running the above command we get some dependency conflicts in
 installed packages which is currently safe to ignore. For details see
 [Issue #71](https://github.com/googlegenomics/gcp-variant-transforms/issues/71).
 
+### Setup IDE
+
+You may choose any IDE as you like. The following steps are intended for
+IntelliJ users.
+
+#### Install IntelliJ IDE
+
+Download
+[IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/download/#section=linux)
+and install.
+
+#### Install Python plugin
+
+Choose File | Settings on the main menu, and then go to Plugins.
+Click the Install JetBrains plugin button.
+In the dialog that opens, search for Python Community Edition and then install
+the plugin.
+
+For more details, refer to
+[Install plugins](https://www.jetbrains.com/help/idea/installing-updating-and-uninstalling-repository-plugins.html).
+
+#### Setup IntelliJ SDK
+
+1. Choose File | Project Structure on the main menu, and then go to Project.
```

Comment:
Please add an initial step on how the project is created. IIRC, the project could have been created in the default location but then there was somewhere that sources were being set, correct?

Reply:
Done. Thanks.